WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SiteCreationPreviewFragment.kt

```diff
@@ -64,22 +64,22 @@ class SiteCreationPreviewFragment : SiteCreationBaseFormFragment(),
     @Inject internal lateinit var viewModelFactory: ViewModelProvider.Factory
     @Inject internal lateinit var uiHelpers: UiHelpers
 
-    private var binding: SiteCreationPreviewScreenBinding? = null
+    private lateinit var binding: SiteCreationPreviewScreenBinding
```

Comment:
By using lateinit var, the need for setting to null is removed. Subsequent binding get() also drop the null requirement. This trickles down to using the fields in the code base.